src/frontend/views/modules/multi-input/multi-input-group-address.pug

```diff
@@ -10,23 +13,18 @@
               ng-options='type as (type | esnI18n) for type in types',
               ng-change="onTypeChange($index)"
             )
-      .col-xs-8.col-sm-9
+      .col-md-1
+        .action-button
+          i.clickable.mdi.mdi-close(ng-show="showDeleteButtonArray[$index]", ng-click="deleteField($index)")
```

Comment:
An icon button should have a hover & wave effect. You can do it by:

```suggestion
          button.btn.btn-link.btn-icon.waves-effect(ng-show="showDeleteButtonArray[$index]", ng-click="deleteField($index)")
            i.mdi.mdi-close
```

The button should have `.all-centered`, width and height of 36px and the icon should have a font size of 20px.